samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.h

```diff
@@ -17,5 +17,30 @@
 
 #import "MenuTableViewController.h"
 
-@interface UILabelViewController : MenuTableViewController
+@interface UILabelViewController : UIViewController <UITextFieldDelegate>
+@property (nonatomic) UITextField* text;
```

Comment:
I remember adding EditingDidEnd or ensuring the keyboard was dismissed but please confirm that it works and meets your needs.

---
In reply to: [105043984](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r105043984) [](ancestors = 105043984,105040618,105038372,104793428)